index.js

```diff
@@ -1 +1,24 @@
+const n = parseInt(
```

Comment:
Looks good! One thing I'd like to point out - Code is not properly formatted. If you are using `VSCode`, please use the `prettier` plugin to reformat the code.

Reply:
Okay sir